services/rds/src/main/java/software/amazon/awssdk/services/rds/RdsPresignInterceptor.java

```diff
@@ -43,6 +44,7 @@
  *
  * @param <T> The request type.
  */
+@SdkInternalApi
```

Comment:
Wonder if this one should be public to allow customers to easily add their own presigners...

Reply:
yeah, updated the class to public